crates/voicevox_core_python_api/python/voicevox_core/_models.py

```diff
@@ -231,6 +242,27 @@ class AudioQuery:
     のAudioQueryでは無視される。
     """
 
+    # `dataclasses.asdict`の内部実装に依存したハックだが、他に方法が思い付かなかった。
+    def __post_init__(self) -> None:
+        """
+        :func:`dataclasses.asdict` にてキーが正しい名前になるよう、 ``dataclass``
+        としてのフィールドをハックする。
+        """
+
+        self.__attr_true_names: dict[str, str] = {}
+        for field in dataclasses.fields(self):
+            if (rename := _rename_audio_query_field(field.name)) != field.name:
+                self.__attr_true_names[rename] = field.name
+                field.name = rename
+
+    def __getattr__(self, name: str) -> object:
+        """camelCaseの名前に対し、対応するsnake_caseの名前があるならそれについて返す。"""
+
+        if true_name := self.__attr_true_names.get(name):
+            return getattr(self, true_name)
+        # 普通の`AttributeError`と同じ文面
+        raise AttributeError(f"{type(self).__name__!r} has no attribute {name!r}")
```

Comment:
多分こういう書き方もあるが、どっちが良いかどうか正直わからない。

```suggestion
        try:
            true_name = self.__attr_true_names[name]
        except KeyError:
            # 普通の`AttributeError`と同じ文面
            raise AttributeError(f"{type(self).__name__!r} has no attribute {name!r}")
        return getattr(self, true_name)
```